packages/firestore/src/remote/rest_connection.ts

```diff
@@ -118,6 +118,7 @@ export abstract class RestConnection implements Connection {
     token: Token | null
   ): void {
     headers['X-Goog-Api-Client'] = X_GOOG_API_CLIENT_VALUE;
+    headers['X-Firebase-GMPID'] = this.databaseInfo.appId;
```

Comment:
Nit: capitalization of the header name is different between this file and `grpc_connection`. Is this intentional?

Reply:
The goal was to match the surrounding code. Let me fix the GRPC headers.